components/01-globals/typography/Label.vue

```diff
@@ -1,19 +1,18 @@
+<template>
+  <component
+    :is="tag ? tag : 'label'"
+    :class="[tag && 'label', {
```

Comment:
why `label` is done in array way, but others in the object, while are doing same thing :D?